[2025-08-13T11:39:47.530+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_pipeline.extract_data manual__2025-08-13T11:39:45.054678+00:00 [queued]>
[2025-08-13T11:39:47.589+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_pipeline.extract_data manual__2025-08-13T11:39:45.054678+00:00 [queued]>
[2025-08-13T11:39:47.593+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-08-13T11:39:47.727+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_data> on 2025-08-13 11:39:45.054678+00:00
[2025-08-13T11:39:47.760+0000] {standard_task_runner.py:57} INFO - Started process 179 to run task
[2025-08-13T11:39:47.768+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_pipeline', 'extract_data', 'manual__2025-08-13T11:39:45.054678+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmp4uzw94vo']
[2025-08-13T11:39:47.783+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask extract_data
[2025-08-13T11:39:47.972+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:193 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-13T11:39:48.220+0000] {task_command.py:416} INFO - Running <TaskInstance: weather_etl_pipeline.extract_data manual__2025-08-13T11:39:45.054678+00:00 [running]> on host 08b9f79dd387
[2025-08-13T11:39:48.601+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-13T11:39:45.054678+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-13T11:39:45.054678+00:00'
[2025-08-13T11:39:50.311+0000] {python.py:194} INFO - Done. Returned value was:    latitude  longitude  ... winddirection_deg  weathercode
0     51.50      -0.12  ...               210            3
1     48.85       2.35  ...               126            2
2     40.71     -74.01  ...               195            0
3     35.68     139.76  ...                84            1
4    -33.87     151.21  ...               215            3

[5 rows x 7 columns]
[2025-08-13T11:39:50.507+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=weather_etl_pipeline, task_id=extract_data, execution_date=20250813T113945, start_date=20250813T113947, end_date=20250813T113950
[2025-08-13T11:39:50.550+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-13T11:39:50.589+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
